### PR TITLE
simbridge: advance model timers

### PR DESCRIPTION
**Pull request by idruzhinin**
Many pipelines rely on timer wheels being advanced accordingly.

### CODE REVIEW

File: hw/pci-bridge/simbridge_utils.h [hunk 1]
```diff
@@ -18,6 +18,7 @@ typedef enum simmsgtype_e {
     SIMMSG_SYNC_REQ,
     SIMMSG_SYNC_ACK,
     SIMMSG_SYNC_REL,
+    SIMMSG_STEP_TIMER_WHEEL,
```
Comment (jbradleysmith) [inline hw/pci-bridge/simbridge_utils.h]: I don't think qemu should be aware of TIMER_WHEELs.  Can we make this just a generic "advance time" message and let the server side turn that into a STEP_TIMER_WHEEL?

Comment (idruzhinin) [reply hw/pci-bridge/simbridge_utils.h]: makes sense, I can push conversion logic lower

File: hw/pci-bridge/simbridge.c [hunk 2]
```diff
@@ -875,6 +875,31 @@ simbridge_poll_for_server(void *opaque)
               qemu_clock_get_ms(QEMU_CLOCK_REALTIME) + 1000);
 }
 
+/* advance model timers by 100us each 1s of vCPU time */
+#define TIMER_DELTA_US 100
```
Comment (jbradleysmith) [inline hw/pci-bridge/simbridge.c]: Even the scale factor is something the server side should handle.  Just give the model a 1s vCPU timer tick msg and let it do the scaling on the model side.

Comment (idruzhinin) [reply hw/pci-bridge/simbridge.c]: I disagree - allowing to advance in different steps is more flexible

Comment (idruzhinin) [reply hw/pci-bridge/simbridge.c]: perhaps I can abstract that out by giving vcpu time to this function

**Review by jbradleysmith — APPROVED** [review]
lgtm, thanks

**Pull request merged**